state-chain/chains/src/arb.rs

```diff
@@ -82,19 +82,26 @@ impl ArbitrumTrackedData {
 	}
 }
 
+pub mod fees {
+	pub const ARB_BASE_COST_PER_BATCH: u128 = 5_200_000;
+	pub const ARB_GAS_COST_PER_FETCH: u128 = 1_700_000;
+	pub const ARB_GAS_COST_PER_TRANSFER_NATIVE: u128 = 1_500_000;
+	pub const ARB_GAS_COST_PER_TRANSFER_TOKEN: u128 = 1_800_000;
+}
```

Comment:
naming nit: since the fees are already namespaced in `arb::fees`, the `ARB_` prefix seems redundant. Same for eth::fees and ETH_.